data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1478 @@
+		log "People" "Anton Kent" "Anton Kent is a Republic official assigned to keep an eye on the systems bordering the Free Worlds. He seems to be unaffiliated with the Navy."
+		log "People" "Amelia Shah" "Amelia Shah is a Republic official assigned to keep an eye on the systems bordering the Free Worlds. She seems to be unaffiliated with the Navy."
+
+
+
+mission "Republic Bounty 3"
+	name "Republic Bounty Hunting"
+	description `Disable the <npc> which seems to be holding merchants for ransom. Do not destroy or attempt to capture the ship. It was last spotted in the Tais system`
+	source
+		government "Republic"
+		attributes "dirt belt"
+	to offer
+		has "Republic Bounty 2: done"
+		not "chosen sides"
+		random < 75
+
+	on offer
+		conversation
+			`As you wander the spaceport, a hand grips your shoulder. You turn to find the surprising strength belongs to Captain Shah. "Captain <last>, the Republic has another job for you. It's different than the last one but I'm sure you'll be interested."`
+			choice
+				`	"Sure, what do I have to do?"`
+				`	"Sorry, I'm a bit busy right now."`
+					defer
+			`	"A ship named the <npc> has captured a number of wealthy merchants and is holding them for ransom. While this should not be a job for an ordinary contractor, there are concerns that they might kill the hostages if approached by a ship flying Republic colors. You will disable the ship and return to me immediately."`
+			choice
+				`	"Consider it done."`
+				`	"I'm sorry, but I can't deal with problems like that."`
+					decline
+			`	Captain Shah gives a small, relieved nod. "The ship was last spotted in Tais. We believe it is still there."`
+				accept
+	npc disable
+		personality heroic staying nemesis target
+		government Bounty
+		system "Tais"
+		ship "Falcon (Heavy)" "Gediminas"
+		dialog `The <npc> has been disabled. You can claim the bounty payment by returning to <destination>.`
+
+	on accept
+		log `Hunting a pirate ship called the "<npc>," which has captured multiple merchant captains. The ship was last sighted in the Tais system.`
+
+	on complete
+		"assisted republic" ++
+		payment 250000
+		dialog `Soon after you land, Captain Shah contacts you. "The hostages were recovered safely," she says, with a smile. "The Republic owes you a debt of gratitude. If you are interested in more tangible benefits, <payment> have been transferred to your account, and I have informed my superiors that you can be counted on."`
+		log "Captain Shah says she has informed her superiors of my contributions."
+
+
+
+mission "Republic Scouting 1"
+	name "Hyperspace Relay Testing"
+	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze the network connectivity of your logs."
+	minor
+	waypoint "Hassaleh"
+	waypoint "Sumar"
+	waypoint "Cardax"
+	waypoint "Volax"
+	source
+		government "Republic"
+		attributes "paradise"
+	destination "Glaze"
```

Comment:
```suggestion
	destination "Farpoint"
```

Reply:
Change destination from a Free Worlds planet, to the north side of the mission waypoints

Reply:
I suspect Glaze was the original choice simply because when this was first raised a while back the concern was made that most new players would end up playing the republic campaign 'by default' simply because they were there, so there was an intention to force people to see other factions before they got locked into anything.
That may not have translated well into actual missions though.

Reply:
Yeah. I don't know why it was chosen to be Glaze. Farpoint makes more sense.